### DISCUSSION TITLE
OSSummit + OSPOCon Japan CFP is now open

### DISCUSSION THREAD

**Discussion by will-ww** [kaiyuanshe/ONES#67] [日常讨论]
OSSummit + OSPOCon Japan CFP is now open: https://events.linuxfoundation.org/open-source-summit-japan/

**Comment by wangleo61** [comment]
王老师，这个是个好东东呀

**Reply by will-ww** [reply]
要不一起申请一个？:-)

**Reply by takasumasakazu** [reply]
我打算帮助这个方面。

**Comment by wangleo61** [comment]
谢谢

从 Windows 版邮件<https://go.microsoft.com/fwlink/?LinkId=550986>发送

发件人: TAKASU ***@***.***>
发送时间: Sunday, April 24, 2022 10:05 PM
收件人: ***@***.***>
抄送: ***@***.***>; ***@***.***>
主题: Re: [kaiyuanshe/ONES] OSSummit + OSPOCon Japan CFP is now open (Discussion #67)


我打算帮助这个方面。

―

**Comment by wangleo61** [comment]
中呀，走一个，咋申请呀，都是日文呀？

从 Windows 版邮件<https://go.microsoft.com/fwlink/?LinkId=550986>发送

________________________________
发件人: Will Wang ***@***.***>
发送时间: Monday, April 4, 2022 7:01:11 PM
收件人: kaiyuanshe/ONES ***@***.***>
抄送: wangleo61 ***@***.***>; Comment ***@***.***>
主题: Re: [kaiyuanshe/ONES] OSSummit + OSPOCon Japan CFP is now open (Discussion #67)


要不一起申请一个？:-)

―
